Compilation with use_openssl fails

I'm using this command in the `bin/` directory: `RUSTFLAGS="-g" CARGO_INCREMENTAL=1 cargo build --release --features "logs-debug,logs-trace,use_openssl"`

I'm using `rustc 1.23.0 (766bd11c8 2018-01-01)`

Errors:
```
error[E0432]: unresolved import `super::https_openssl::configuration`
  --> /root/sozu/lib/src/network/mod.rs:31:33
   |
31 |   pub use super::https_openssl::configuration::start;
   |                                 ^^^^^^^^^^^^^ Could not find `configuration` in `https_openssl`

error[E0433]: failed to resolve. Use of undeclared type or module `https_openssl`
  --> /root/sozu/lib/src/network/mod.rs:28:34
   |
28 |   pub type ServerConfiguration = https_openssl::ServerConfiguration;
   |                                  ^^^^^^^^^^^^^ Use of undeclared type or module `https_openssl`

error[E0433]: failed to resolve. Use of undeclared type or module `https_openssl`
  --> /root/sozu/lib/src/network/mod.rs:29:34
   |
29 |   pub type TlsClient           = https_openssl::TlsClient;
   |                                  ^^^^^^^^^^^^^ Use of undeclared type or module `https_openssl`

error: aborting due to 3 previous errors

error: Could not compile `sozu-lib`.
```

--- a/lib/src/network/mod.rs
+++ b/lib/src/network/mod.rs
@@ -25,10 +25,10 @@ pub mod https_openssl;
 
 #[cfg(feature = "use_openssl")]
 pub mod https {
-  pub type ServerConfiguration = https_openssl::ServerConfiguration;
-  pub type TlsClient           = https_openssl::TlsClient;
+  pub type ServerConfiguration = super::https_openssl::ServerConfiguration;
+  pub type TlsClient           = super::https_openssl::TlsClient;
 
-  pub use super::https_openssl::configuration::start;
+  pub use super::https_openssl::start;
 }
 
 #[cfg(not(feature = "use_openssl"))]

